docs/upload_mapping_file.md

```diff
@@ -136,6 +136,26 @@ For example:
 tasks["minify${variant}WithR8"].finalizedBy { tasks["uploadMapping${variant}"] }
 ```
 
+## Limitations
+
+For now, we can only upload mapping files with a size smaller than 50 Mb. If your project has a mapping file larger
+than 50Mb, you can try several options to reduce the file size and be able to upload it.
```

Comment:
Mapping files are limited to 50 Mb. If your project has a mapping file larger than 50 Mb, use one of the following options to reduce the file size: